### PR TITLE
Add new and complete documentation

### PR DESCRIPTION
**Pull request by shreyasminocha**
Supposed to be a replacement for the docs in "Wiki" and the JSDocs toolchain. We could probably host a GH pages site from `docs/`.

screenshot 2018-08-14 at 7 58 59 pm

### CODE REVIEW

File: VerbalExpressions.js [hunk 1]
```diff
@@ -76,66 +51,30 @@ class VerbalExpression extends RegExp {
...
         value = VerbalExpression.sanitize(value);
         return this.add(`(?:${value})`);
     }
 
-    /**
-     * Alias for then() to allow for readable syntax when then() is the first method in the chain.
-     * @param {(string|RegExp|numer)} value value to find
-     * @returns {VerbalExpression} recompiled instance of VerbalExpression
-     * @memberof VerbalExpression
-     */
```
Comment (ionutvmi) [inline VerbalExpressions.js]: can we still keep the jsdoc comments here even if they are not used for generating the /docs ?

Comment (shreyasminocha) [reply VerbalExpressions.js]: If you guys insist, sure.

Comment (shreyasminocha) [reply VerbalExpressions.js]: @ionutvmi done.

**Comment by ionutvmi** [comment]
It needed a few adjustments but the site is now running at: http://verbalexpressions.github.io/JSVerbalExpressions/

Thank you for your contribution !

**Pull request merged**